src/metafiles/cvs.rs

```diff
@@ -1,19 +1,30 @@
 use std::{borrow::Cow, fmt, str::FromStr};
 
 use miette::Diagnostic;
-use once_cell::sync::Lazy;
 use semver::Version;
 use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use thiserror::Error;
 use url::Url;
 
+use crate::cratesio_url;
+
 #[derive(Clone, Debug, Ord, PartialOrd, Eq, PartialEq)]
 pub struct CrateVersionSource {
     pub name: String,
     pub version: Version,
     pub source: Source,
 }
 
+impl From<&super::binstall_v1::MetaData> for CrateVersionSource {
+    fn from(metadata: &super::binstall_v1::MetaData) -> Self {
+        super::CrateVersionSource {
+            name: metadata.name.clone().to_string(),
```

Comment:
CompactString implements `Display`, so `metadata.name.to_string()` should be sufficient

Reply:
Also could we use `CompactString` in the cvs struct instead?

Reply:
> Also could we use CompactString in the cvs struct instead?

Done!